README.md

```diff
@@ -67,7 +89,7 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
-[] Selects all ul's of class dropdown, only if their children are exclusively li's
+[x] Selects all ul's of class dropdown, only if their children are exclusively li's
```

Comment:
The answer is the first choice. There actually isn't a way to do this choice in CSS: you can't select an element based on its descendants, only based on its ancestors.